CONTRIBUTING.md

```diff
@@ -56,7 +56,7 @@ ask for respect and understanding and will try to provide the same back.
 
 Before requesting or implementing a new feature, please do the following:
 
-- Take a look at our [list of plugins](http://www.rebar3.org/docs/using-available-plugins)
+- Take a look at our [list of plugins](https://rebar3.readme.io/docs/using-available-plugins#available-plugins)
```

Comment:
```suggestion
- Take a look at our [list of recommended plugins](http://rebar3.org/docs/configuration/plugins/#recommended-plugins)
```

We moved off of readme.io so this is the new link.

Reply:
Thanks, I updated the link.